pkg/bgpv1/README.md

```diff
@@ -1,5 +1,14 @@
 # Cilium BGP Control Plane
 
+BGP Control Plane provides a way for Cilium to advertise routes to Cilium-managed endpoints by
```

Comment:
Same comments as above in this section as well.

Reply:
Mine as well. ☝🏻